### PR TITLE
Fixing issues when naming structs and adding package name with semi-colon

### PR DESCRIPTION
**Pull request by NatashaWso2**
Fixes https://github.com/ballerinalang/composer/issues/3514

### CODE REVIEW

File: modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx [hunk 1]
```diff
@@ -95,6 +95,10 @@ class TopLevelNodes extends React.Component {
             this.props.model.viewState.packageDefExpanded = false;
         }
         if (this.packageDefValue) {
+            // If there is a semi-colon at the end of the package name, remove it
+            if (this.packageDefValue.slice(-1) === ';') {
```
Comment (hemikak) [inline modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx]: You can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith function here.

Comment (hemikak) [inline modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx]: Or use this.packageDefValue = _.trimEnd(this.packageDefValue, ';') , so  you can get rid of this whole block.

Comment (NatashaWso2) [reply modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx]: Used the _trimEnd function of Lodash  https://lodash.com/docs/4.17.4#trimEnd 
Fixed with commit 05f4d18

Comment (NatashaWso2) [reply modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx]: Fixed with commit 05f4d18

**Pull request merged**